### PR TITLE
Update rules pop-up

### PR DESCRIPTION
**Pull request by YuriyLu**

### CODE REVIEW

File: app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx [hunk 1]
```diff
@@ -2,13 +2,18 @@ import React, { useRef, useState, RefObject } from 'react';
...
+import { singleCommunityColors } from 'utils/communityManagement';
 
 import commonMessages from '../../../common-messages';
 import { styles } from './AskQuestionPopup.styled';
+import LargeButton from 'components/Button/Contained/InfoLarge';
 
-const stylesCommunity = singleCommunityStyles();
+const colors = singleCommunityColors();
+
+const Button = LargeButton.extend`
```
Comment (Lampardks) [inline app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx:13]: use emotion

File: app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx [hunk 2]
```diff
@@ -41,56 +46,80 @@ const AskQuestionPopup: React.FC = (): JSX.Element => {
...
-                  <FormattedMessage
-                    id={commonMessages.rulesAskQuestionPopupBlock_4.id}
-                  />
-                </div>
-                <div>
-                  <FormattedMessage
-                    id={commonMessages.rulesAskQuestionPopupBlock_5.id}
-                  />
-                </div>
+              <div1111 className="df fdc jcsb pt24 pb24 lh1-5">
```
Comment (Lampardks) [inline app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx:49]: why div1111 ?

File: app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx (hunk 1 above)
Comment (YuriyLu) [reply app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx]: done

File: app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx (hunk 2 above)
Comment (YuriyLu) [reply app/containers/AskQuestion/AskQuestionPopup/AskQuestionPopup.tsx]: fixed

**Pull request merged**